tests/unit/pyrflx_test.py

```diff
@@ -1356,3 +1368,40 @@ def test_get_path(icmp_message_value: MessageValue) -> None:
 
 def test_get_model(icmp_message_value: MessageValue) -> None:
     assert isinstance(icmp_message_value.model, Message)
+
+
+def test_parameterized_message(parameterized_package: Package) -> None:
+    message = parameterized_package.new_message(
+        "Message", {"Length": 8, "Has_Tag": False, "Tag_Value": "Tag_A"}
+    )
+    assert message.fields == ["Payload", "Tag"]
+    assert message.required_fields == ["Payload"]
+    message.set("Payload", bytes(8))
+    assert message.required_fields == []
+    assert message.valid_message
+    assert message.bytestring == bytes(8)
+
+
+def test_parameterized_message_no_verification() -> None:
+    pyrflx_ = PyRFLX.from_specs(
+        [SPEC_DIR / "parameterized.rflx"],
+        skip_model_verification=True,
+        skip_message_verification=True,
+    )
+    message_unv = pyrflx_.package("Parameterized").new_message(
+        "Message", {"Length": 8, "Has_Tag": False, "Tag_Value": "Tag_A"}
+    )
+    assert message_unv.fields == ["Payload", "Tag"]
+    message_unv.set("Payload", bytes(8))
+    assert message_unv.valid_message
+    assert message_unv.bytestring == bytes(8)
+
+
+def test_parameterized_message_invalid_type(parameterized_package: Package) -> None:
+    with pytest.raises(
+        PyRFLXError, match=f"^pyrflx: error: {type(bytes())} is no supported parameter type"
```

Comment:
Error message should be matched completely (missing `$`).

Reply:
Fixed in https://github.com/Componolit/RecordFlux/pull/778/commits/c09d5c88c54b4124bb4fa6cf838bef01ae2e6633.